### ISSUE TITLE
About pre-train model

### ISSUE THREAD

**Issue by CheungBH** [huawei-noah/GAN-pruning#19]
Hello. I am using the ManiDP code, and I want to train with Cifar10. However, I don't know where to find the pre-train model. Could you provide the pre-train model or the code for original training?

**Comment by CheungBH** [comment]
Could I just get the pre-train model by assigning  "--target_remain_rate=1 --lambda_lasso 0 --lambda_graph 0"?

**Comment by CheungBH** [comment]
> Could I just get the pre-train model by assigning "--target_remain_rate=1 --lambda_lasso 0 --lambda_graph 0"?

Failed. The loss is also nan

**Comment by yehuitang** [comment]
Thanks for your interesting! There is no channel selection module when obtaining the pre-trained models.
